MementoMori/mementomori.client/src/deckBrowser/TagSelector.tsx

```diff
@@ -154,6 +154,7 @@ const Listbox = styled('ul')(
 );
 
 export default function TagSelector(props: {
+  selectedTags: string[]; // Add selectedTags prop
   setSelectedTags: React.Dispatch<React.SetStateAction<string[]>>;
 }) {
```

Comment:
Do other places where `TagSelector` is used still work after the change of its signature?